Assets/Unity Simple Liquid/Scripts/SplitController.cs

```diff
@@ -269,9 +269,10 @@ private RaycastHit FindLiquidContainer(Vector3 splitPos, GameObject ignoreCollis
 					var liquid = hit.collider.GetComponent<SplitController>();
 					if (liquid && liquid != this)
 					{
-						
+						//color change in capacity
+						SendLiquidContainer(hit.collider.gameObject);
+
```

Comment:
This line can change color, even when liquids flow doesn't touch bottleneck. For example when bottle is upside down. I would recommend move it inside TransferLiquid function.
```
if (insideRadius)
{
	var lostAmount = liquidContainer.Volume * lostPercentAmount;
	liquid.liquidContainer.FillAmount += lostAmount;


	//color change in capacity
        SendLiquidContainer(hit.collider.gameObject);
}
```

Reply:
Good